www/templates/keypad.html

```diff
@@ -0,0 +1,20 @@
+<div class="calc-row button-bar bar-stable">
+  <div class="button" ng-click="keyPress(1)">1</div>
+  <div class="button" ng-click="keyPress(2)">2</div>
+  <div class="button" ng-click="keyPress(3)">3</div>
+</div>
+<div class="calc-row button-bar bar-light">
+  <div class="button" ng-click="keyPress(4)">4</div>
+  <div class="button" ng-click="keyPress(5)">5</div>
+  <div class="button" ng-click="keyPress(6)">6</div>
+</div>
+<div class="calc-row button-bar bar-stable">
+  <div class="button" ng-click="keyPress(7)">7</div>
+  <div class="button" ng-click="keyPress(8)">8</div>
+  <div class="button" ng-click="keyPress(9)">9</div>
+</div>
+<div class="calc-row button-bar bar-light">
+  <div class="button" ng-click="clearEntry()">c</div>
+  <div class="button" ng-click="keyPress(0)">0</div>
+  <div class="button" ng-click="addHundred()">00</div>
```

Comment:
Seems like this should be a `keyPress` call here too since it's really a key press. I suggest you pass keypress a string instead of an integer, then you can pass '00', and then decide how to handle it in the controller. addNumber and addHundred should really be private methods on the controller. This reduces coupling.